deploy/README.md

```diff
@@ -126,26 +126,22 @@ Next step is to configure your resources and set them up with the required depen
 
 ## Packaging the Synapse Pipeline
 
-You may repeat the steps outlined in this section multiple times to package the pipeline irrespective of whether you have already run the `package.sh` script or `setup.sh` script before. 
+You may repeat the steps outlined in this section multiple times to package the pipeline irrespective of whether you have already run the `package.sh` script or `setup.sh` script. 
 
 To package the Synapse pipeline, run the `package.sh` script by following the syntax below:
 
 ```bash
 ./deploy/package.sh <environmentCode> <pipelineName>
 ```
+Unzip the contents of the ZIP file and use the contents to load them to your repository.
 
-Once the above step completes, a zip file is generated. Upload the generated zip files to your Synapse Studio by following the steps below:
+**Note**: Do not bring the contents of the pipeline from [workflow folder](../src/workflow/custom-vision-model-v2) directly into your repository. You will need to run `package.sh` or `setup.sh` to replace placeholder in one or more files before checking-in the files to your repository.
```

Comment:
Rest changes are looking good. This note is not coming out clearly from a user point of view. Need to re phrase

Reply:
Addressed this feedback.